sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/datasources/subscriptions/SubscriptionsDbConnector.groovy

```diff
@@ -133,4 +133,19 @@ class SubscriptionsDbConnector implements SubscriptionDataSource {
             }
         return isAlreadySubscribed
     }
+
+    @Override
+    List<String> findSubscribedProjectCodesFor(Subscriber subscriber) {
+        List<String> subscribedProjects = []
+        int subscriberId = fetchExistingSubscriberId(subscriber)
```

Comment:
subscriberId can be -1 (nonexisting). Does your query return an empty List then or may it cause an exception?

Reply:
whaaa ...`-1` ?